services/scan/src/interpreter.ts

```diff
@@ -63,7 +63,7 @@ interface BallotImageData {
 export async function getBallotImageData(
   file: Buffer,
   filename: string,
-  detectQrcodeResult: qrcodeWorker.Output
+  detectQrcodeResult: qrcodeWorker.NonBlankPageOutput
```

Comment:
this function only gets called on L238, so I was able to narrow the type after adding the new guard on L234